src/client/javascripts/application.js

```diff
@@ -0,0 +1,21 @@
+import {
+  createAll,
+  Button,
+  CharacterCount,
+  Checkboxes,
+  ErrorSummary,
+  Header,
+  NotificationBanner,
+  Radios,
+  SkipLink
+  // @ts-expect-error -- No types available
+} from 'govuk-frontend'
+
+createAll(Button)
+createAll(CharacterCount)
+createAll(Checkboxes)
+createAll(ErrorSummary)
+createAll(Header)
+createAll(NotificationBanner)
+createAll(Radios)
+createAll(SkipLink)
```

Comment:
Feels a bit risky given how much we use, without testing the whole runner. Is there much of a cost saving vs just running initAll?

Reply:
Yeah over half the file size comes from unused components

I've followed the [**Import individual components** guidance](https://frontend.design-system.service.gov.uk/import-javascript/#import-individual-components) and [contributed it to CDP recently](https://github.com/DEFRA/cdp-node-frontend-template/pull/80)

Otherwise we'd bundle and push out these components out unnecessarily:

* [Accordion](https://govuk-frontend-review.herokuapp.com/docs/stats/components/accordion/accordion.html) 23.5 KiB
* [Exit this page](https://govuk-frontend-review.herokuapp.com/docs/stats/components/exit-this-page/exit-this-page.html) 17.1 KiB
* [Password input](https://govuk-frontend-review.herokuapp.com/docs/stats/components/password-input/password-input.html) 15.15 KiB
* [Tabs](https://govuk-frontend-review.herokuapp.com/docs/stats/components/tabs/tabs.html) 10.13 KiB

You can see all file sizes here if you scroll down to **JavaScript modules**
https://govuk-frontend-review.herokuapp.com

<img width="689" alt="File sizes" src="https://github.com/DEFRA/forms-runner/assets/415517/54c5883b-2e7c-48a0-8a06-7f016d384ea9">

Reply:
Nice writeup, thanks! Looks good then